tests/unit/index-nodetest.js

```diff
@@ -454,4 +454,45 @@ describe('redis plugin', function() {
       assert.match(messageOutput, /ember deploy:activate qa --revision=123abc/);
     });
   });
+
+  describe('fetchRevisions hook', function() {
+    it('fills the revisions variable on context', function() {
+      var plugin;
+      var context;
+
+      plugin = subject.createDeployPlugin({
+        name: 'redis'
+      });
+
+      context = {
+        ui: mockUi,
+        project: stubProject,
+        config: {
+          redis: {
+            keyPrefix: 'test-prefix',
+            filePattern: 'index.html',
+            distDir: 'tests',
+            revisionKey: '123abc',
+            redisDeployClient: function(context) {
```

Comment:
Also, just wondering if we can use the FakeRedis client here instead of overriding the redisDeployClient property?